models/bootcampers.js

```diff
@@ -0,0 +1,41 @@
+import { pool } from '../db/index.js'
+
+export async function getAllBootcampers () {
+  const sqlQuery =
+    'SELECT * FROM bootcampers'
+  const result = await pool.query(sqlQuery)
+  const bootcampers = result.rows
+  return bootcampers
+}
+
+export async function getBootcamperByID (id) {
+  const data = await pool.query(
+    'SELECT * FROM bootcampers WHERE id = $1',
+    [id]
+  )
+  return data.rows[0]
+}
+
+export async function createUser (newUser) {
+  const data = await pool.query(
+    'INSERT INTO bootcampers (username, is_coach) VALUES ($1, $2) RETURNING *;',
+    [newUser.username, newUser.is_coach]
+  )
+  return data.rows[0]
+}
+
+export async function updateBootcamper (id, updatedBootcamper) {
+  const data = await pool.query(
```

Comment:
Do we need to have a check here to make sure the bootcamper exists before updating?